### ISSUE TITLE
2FA Challenge

### ISSUE THREAD

**Issue by Najib90** [joshiayush/inb#72]
After running the tool in Macbook M1 I get the following error as my Linkedin is set up with 2-factor authentication:

```
raise linkedin_api_exceptions.LinkedInChallengeException(
api.exceptions.LinkedInChallengeException: CHALLENGE
```

Any way around this?

**Comment by joshiayush** [comment]
Yeah, that's because it can't login. Try removing 2FA and see if it works!

**Comment by Najib90** [comment]
> Yeah, that's because it can't login. Try removing 2FA and see if it works!

It still didnd't work, it seems the code is using outdated APIs?

**Comment by yesbhautik** [comment]
> > Yeah, that's because it can't login. Try removing 2FA and see if it works!
> 
> It still didnd't work, it seems the code is using outdated APIs?

That's true.

**Comment by joshiayush** [comment]
Unfortunately, this project isn't maintained at all, old APIs might have given up by now.
